dts/bindings/i2c/nxp,sc18im704-i2c.yaml

```diff
@@ -0,0 +1,16 @@
+# Copyright (c), 2023 Basalte bv
+# SPDX-License-Identifier: Apache-2.0
+
+description: I2C controller part for the SC18IM704 bridge
+
+compatible: "nxp,sc18im704-i2c"
+
+include: i2c-controller.yaml
+
+properties:
+  verify:
```

Comment:
What is the intent of this feature?  Wondering if this should be a kconfig option instead.

Reply:
The only way to know if a write succeeded is to read a status register from the chip. 
A kconfig option would set this for all instances. I can change if this is preferred.

Reply:
I guess the question is when would someone want to enable `verify`.  I'd say maybe lets have it as a Kconfig option and if someone comes up w/a need for it to truly be per-instance we can shift it to a devicetree property than, since this feels like something you'd either always have on or off if you had multiple instances.